From 2a1cca399be415d6c5a556af2018e5fb726d9a37 Mon Sep 17 00:00:00 2001
From: "H.J. Lu" <hjl.tools@gmail.com>
Date: Fri, 1 Apr 2016 14:01:24 -0700
Subject: [PATCH 25/25] X86-64: Add dummy memcopy.h and wordcopy.c

Since x86-64 doesn't use memory copy functions, add dummy memcopy.h and
wordcopy.c to reduce code size.  It reduces the size of libc.so by about
1 KB.

	* sysdeps/x86_64/memcopy.h: New file.
	* sysdeps/x86_64/wordcopy.c: Likewise.
---
 sysdeps/x86_64/memcopy.h  | 1 +
 sysdeps/x86_64/wordcopy.c | 1 +
 2 files changed, 2 insertions(+)
 create mode 100644 sysdeps/x86_64/memcopy.h
 create mode 100644 sysdeps/x86_64/wordcopy.c

diff --git a/sysdeps/x86_64/memcopy.h b/sysdeps/x86_64/memcopy.h
new file mode 100644
index 0000000..590b6cb
--- /dev/null
+++ b/sysdeps/x86_64/memcopy.h
@@ -0,0 +1 @@
+/* X86-64 doesn't use memory copy functions.  */
diff --git a/sysdeps/x86_64/wordcopy.c b/sysdeps/x86_64/wordcopy.c
new file mode 100644
index 0000000..590b6cb
--- /dev/null
+++ b/sysdeps/x86_64/wordcopy.c
@@ -0,0 +1 @@
+/* X86-64 doesn't use memory copy functions.  */
-- 
2.8.1

